app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    splitted_cmd = command.split()
+    if len(splitted_cmd) != 3:
+        return
+    if splitted_cmd[0] != "mv":
+        return
+    cmd, file_from, file_to = splitted_cmd
+    if "/" in file_to or r"\\" in file_to:
```

Comment:
You can use os.path.split which will split the path to 2 parts: directories and filename. If the first part is not empty it means that the path contains directories. So you can avoid this manual check of "/" and "\"